eth/backend.go

```diff
@@ -641,16 +643,53 @@ var (
 // StartCheckpointWhitelistService starts the goroutine to fetch checkpoints and update the
 // checkpoint whitelist map.
 func (s *Ethereum) startCheckpointWhitelistService() {
+	// first run the checkpoint whitelist
+	firstCtx, cancel := context.WithTimeout(context.Background(), whitelistTimeout)
+	err := s.handleWhitelistCheckpoint(firstCtx)
+
+	cancel()
+
+	if err != nil {
+		if errors.Is(err, ErrBorConsensusWithoutHeimdall) || errors.Is(err, ErrNotBorConsensus) {
```

Comment:
I see this pattern quite often. Could we try to introduce in `common` package something like

```
func AnyError(src error, errs ...error) bool {
	for _, err := range errs {
		if errors.Is(src, err) {
			return true
		}
	}

	return false
}
``` 
?